salt/metalk8s/defaults.yaml

```diff
@@ -3,6 +3,9 @@ cluster: kubernetes
 
 metalk8s: {}
 
+kubernetes:
+  version: 1.11.7  # NOTE: remember to update kubelet version as well
```

Comment:
You will need to change this in integration branches no?

Reply:
Yes I'm actually doing it in the merge commits